tests/test_align.py

```diff
@@ -79,7 +82,9 @@ def test_align_47tuc(self):
         rms_x = max(shift_file['col6'])
         rms_y = max(shift_file['col7'])
 
-        assert (rms_x <= 0.25 and rms_y <= 0.25)
+        reference_wcs = amutils.build_reference_wcs(input_filenames)
```

Comment:
Looks like `input_filenames` needs to be set

Reply:
That is what happens when you copy-paste and overlook the differences in each test definition.  This has been corrected in 57113bb